test/cppsuite/how_to_use_cppsuite.md

```diff
@@ -0,0 +1,175 @@
+        value_size_bytes = value_size_bytes == 1000 ? 10000000 : 1000;
+
+        tw->txn.begin();
+
+        bool success = true;
+        // Each transaction should have target_op_count insert operations. This value is set in the configuration file
+        for(int i = 0; i < tw->txn.get_target_op_count(); ++i) {
+            tw->sleep();
+    
+            // Populate the key and value with random strings. We only use the size of the value in this test
+            const std::string key = random_generator::instance().generate_pseudo_random_string(tw->key_size);
+            const std::string value = random_generator::instance().generate_pseudo_random_string(value_size_bytes);
+
+            if(!tw->insert(cursor, coll.id, key, value)) {
+                success = false;
+                break;
+            }
+        }
+
+        if(!success){
+            tw->txn.rollback();
+        } else {
+            testutil_assert(tw->txn.commit());
+        }
+    }
+}
+```
+
+
+#### Saving validation data
+The `set_tracking_cursor` function defines what data is saved by the operation tracker. 
+In our example test we will save the timestamp, transaction ID, key and value size.
+
+```cpp
+void
+set_tracking_cursor(WT_SESSION *session, const tracking_operation &operation, const uint64_t &collection_id,
+    const std::string &key, const std::string &value, wt_timestamp_t ts,
+    scoped_cursor &op_track_cursor) override final
+{
+    uint64_t txn_id = ((WT_SESSION_IMPL *)session)->txn->id;
+
+    op_track_cursor->set_key(op_track_cursor.get(), ts, txn_id);
+    op_track_cursor->set_value(op_track_cursor.get(), key.c_str(), value.size());
+}
+```
+*Note: We also need to specify the key-value format of the tracking table where we save test data. This is covered [below](#the-configuration-file).*
+
+##### Performing validation
+When the test finishes the framework calls the `validate` function in which users can perform any checks they want. 
+
+In our test `multi_size_inserts` the 10MB updates are too large for cache and not be written ([see below](#the-configuration-file) for how to configure cache size). As such we will check that any update made with a value size of 1KB is present in the database, and any update with a value size of 10MB is not present:
+
+```cpp
+void
+validate(const std::string &operation_table_name, const std::string &schema_table_name,
+    database &db) override final
+{
+    scoped_session session = connection_manager::instance().create_session();
+
+    // For this test we only need to check the insert operations data saved by the operation tracker.
+    scoped_cursor tracked_operations_cursor = session.open_scoped_cursor(operation_table_name);
+    testutil_assert(!db.get_collection_ids().empty());
+    scoped_cursor cursor = session.open_scoped_cursor(db.get_collection(0).name);
+
+    while (tracked_operations_cursor->next(tracked_operations_cursor.get()) == 0) {
+        int ts, txn_id, value_size;
```

Comment:
As a precaution, it's good practice to initialise all variables.

Reply:
Updated in a1ee228bf1c057be8095493025bd6ae21c61d804

Reply:
Thanks!